components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/util/OIDCSessionManagementUtil.java

```diff
@@ -278,22 +278,22 @@ public static boolean getOpenIDConnectSkipeUserConsent() {
         return OAuthServerConfiguration.getInstance().getOpenIDConnectSkipeUserConsentConfig();
     }
 
-    private static String generateSaltValue() throws NoSuchAlgorithmException {
-
-        byte[] bytes = new byte[16];
-        SecureRandom secureRandom = SecureRandom.getInstance(RANDOM_ALG_SHA1);
-        secureRandom.nextBytes(bytes);
-        return Base64.encodeBase64URLSafeString(bytes);
-    }
-
-    private static String bytesToHex(byte[] bytes) {
-
-        StringBuilder result = new StringBuilder();
-        for (byte byt : bytes) {
-            result.append(Integer.toString((byt & 0xff) + 0x100, 16).substring(1));
-        }
-        return result.toString();
-    }
+//    private static String generateSaltValue() throws NoSuchAlgorithmException {
```

Comment:
why commented code?

Reply:
Since these private methods are not used, findbugs reported as an issue.

Reply:
If they are not used, please remove them.

Reply:
Removed